README.md

```diff
@@ -146,3 +164,20 @@ These are visual aids that help tell the story of why Multihash matters.
 #### Multihash: has a bunch of implementations already
 
 ![](https://raw.githubusercontent.com/multiformats/multiformats/eb22cd807db692877a9094b5bfb4d2997fd0278a/img/multihash.008.jpg)
+
+## Maintainers
+
+Captain: [@jbenet](https://github.com/jbenet).
+
+## Contribute
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/multihash/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+
+## License
+
+[MIT](LICENSE) © Protocol Labs Inc.
```

Comment:
- sometimes the copyright notice is longer than one line, or it's complicated. or it's socially contentious.
- would be GREAT not to have to show this here and defer to the `LICENSE` file, is there a reason you want it here?